### PR TITLE
Allow running a single test file

### PR DESCRIPTION
**Pull request by kmcphillips**
@byroot @jules2689 

Allows a single test file to be run using `dev test test/file_test.rb`

### CODE REVIEW

File: dev.yml [hunk 1]
```diff
@@ -16,7 +16,16 @@ commands:
   bootstrap: bin/bootstrap shopify
   console: test/dummy/bin/rails console
   server: test/dummy/bin/rails server -b 192.168.64.1 -p 55330
-  test: bin/rake
```
Comment (casperisfine) [inline dev.yml]: Now that shipit-engine is on Rails 5, you can simply invoke `bin/rails test "$@"`, no need for conditionals.

Comment (kmcphillips) [inline dev.yml]: Sure!

**Comment by kmcphillips** [comment]
Well that's way cleaner.

**Comment by casperisfine** [comment]
Thanks.

**Pull request merged**